core/domain/wipeout_domain.py

```diff
@@ -86,15 +79,3 @@ def create_default(
         """
         return cls(
             user_id, email, normalized_long_term_username, False, {})
-
-    def validate(self) -> None:
```

Comment:
@vojtechjelinek One question for you -- I understand why we're removing this (from the imports perspective), but doesn't this break the pattern of "each domain object should have its own validate method"? There are alternative solutions like pulling out models.NAMES into a constants file somewhere, so I just wanted to check with you whether, given that some of the generic infrastructure is changing, we are intending to impose any general constraints on validate() (e.g. each domain object must have a validate() method) and whether this PR is still in line with that.

Reply:
Ah, I didn't realize this. Yeah, I think the best solution would be to move the `NAMES` into the _feconf.py_ if we want to keep the `validate` function in the domain object. Right now it is not required to have `validate` in all domain objects, but it certainly makes sense here.

Reply:
Done

Reply:
Done

Reply:
Done

Reply:
Done.

Reply:
Done.

Reply:
Not fully done, right? We can now keep using the function here.